airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+DEFAULT_FORMAT_IGNORE_LIST = [
+    "**/__pycache__",
+    '"**/.pytest_cache',
+    "**/.venv",
+    "**/venv",
+    "**/.gradle",
+    "**/node_modules",
+    "**/.tox",
+    "**/.eggs",
+    "**/.mypy_cache",
+    "**/.venv",
+    "**/*.egg-info",
+    "**/build",
+    "**/dbt-project-template",
+    "**/dbt-project-template-mssql",
+    "**/dbt-project-template-mysql",
+    "**/dbt-project-template-oracle",
+    "**/dbt-project-template-clickhouse",
+    "**/dbt-project-template-snowflake",
+    "**/dbt-project-template-tidb",
+    "**/dbt-project-template-duckdb",
+    "**/dbt_test_config",
+    "**/normalization_test_output",
+    # '**/tools',
+    "**/secrets",
+    "**/charts",  # Helm charts often have injected template strings that will fail general linting. Helm linting is done separately.
```

Comment:
I didn't curate this list as well as I should have and I'm pretty sure we don't have helm charts in this repo. Just a FYI, not a blocking comment.